docs/administration/retention-policies/index.md

```diff
@@ -39,9 +39,17 @@ We talk about Tentacles here, but the same process and logic applies to [SSH Tar
 
 ### Built-in repository {#built-in-repo-whats-deleted}
 
-The built-in repository will delete any **packages** that are not attached to any release. If you happen to have higher versions of packages that have not been released, we will keep them assuming a release will be created. If you delete releases using the Octopus Server retention policy then any packages that were associated with those releases will then be deleted with that task.
+A retention policy can be applied to packages in the built-in Octopus package repository. By default, the policy is set to keep all packages indefinitely. This policy is *separate* from the [release retention policy](#releases-whats-deleted) described above.
 
-A list of packages IDs that a project has deployed is kept and then used to determine retention for projects that [dynamically select packages using variables](/docs/deployments/packages/dynamically-selecting-packages.md). A package will be kept if it appears in that list and the package's version matches any of the package versions referenced by the project's releases.
```

Comment:
I chose to remove these lines as well because I don't believe they contribute to the user's goal of understanding the behavior of the retention policy. How Octopus does the work behind the scenes might be interesting, but probably not useful to the reader.

Reply:
I agree, the inner workings of the product for the most part are more likely noise, particularly on this page which is already a complicated set of information to digest.